### PR TITLE
Update Quantity and Time preview

### PR DESCRIPTION
**Pull request by Vesihiisi**
If the claim value is a quantity, output: `<quantity_value> {{Q|<unite_qid>}}`
If the claim value is a date, output it in iso-format, eg. `1999-12-12`

Preview page with these enabled: https://www.wikidata.org/wiki/User:Alicia_Fagerving_(WMSE)/sandbox2

Task: https://phabricator.wikimedia.org/T160334

### CODE REVIEW

File: importer/Monument.py [hunk 1]
```diff
@@ -62,7 +62,15 @@ def print_wd_to_table(self):
                 value_to_print = ""
                 if utils.string_is_q_item(value):
                     value = utils.wd_template("Q", value)
-                value_to_print = value_to_print + str(value)
+                    value_to_print = value_to_print + str(value)
```
Comment (lokal-profil) [inline importer/Monument.py]: use += instead

File: importer/importer_utils.py [hunk 2]
```diff
@@ -387,6 +387,17 @@ def date_to_dict(datestring, dateformat):
     return date_dict
 
 
+def dict_to_iso_date(date_dict):
```
Comment (lokal-profil) [inline importer/importer_utils.py]: Add docstring

File: importer/Monument.py (hunk 1 above)
Comment (Vesihiisi) [inline importer/Monument.py]: Done.

File: importer/importer_utils.py (hunk 2 above)
Comment (Vesihiisi) [inline importer/importer_utils.py]: Done.

**Comment by lokal-profil** [comment]
Linebreak the new docstring otherwise it looks good

File: importer/importer_utils.py [hunk 3]
```diff
@@ -392,7 +392,8 @@ def dict_to_iso_date(date_dict):
     Convert pywikiboty-style date dictionary
     to ISO string ("2002-10-23").
 
-    @param date_dict: dictionary like {"year" : 2002, "month" : 10, "day" : 23}
+    @param date_dict: dictionary like
+    {"year" : 2002, "month" : 10, "day" : 23}
```
Comment (lokal-profil) [inline importer/importer_utils.py]: I would indent this once

**Pull request merged**